src/ring_theory/jacobson.lean

```diff
@@ -116,4 +121,156 @@ lemma is_jacobson_iso (e : R ≃+* S) : is_jacobson R ↔ is_jacobson S :=
 
 end is_jacobson
 
+
+section localization
+open localization_map
+variables {y : R} (f : localization_map (submonoid.closure {y} : submonoid R) S)
+
+lemma disjoint_closure_singleton_iff_not_mem {I : ideal R} (hI : I.radical = I) :
+  disjoint ((submonoid.closure {y} : submonoid R) : set R) ↑I ↔ y ∉ I.1 :=
+begin
+  refine ⟨λ h, set.disjoint_left.1 h (submonoid.subset_closure (by simp)), λ h, _⟩,
+  rw disjoint_iff,
+  rw eq_bot_iff,
+  rintros x ⟨hx, hx'⟩,
+  exfalso,
+  apply h,
+  erw submonoid.mem_closure_singleton at hx,
+  cases hx with n hn,
+  rw ← hn at hx',
+  rw ← hI at hx' ⊢,
+  refine mem_radical_of_pow_mem hx',
+end
+
+/-- The submonoid generated by the single element `y` -/
+abbreviation M (y : R) : submonoid R := (submonoid.closure {y} : submonoid R)
```

Comment:
In general, we don't really make such 1 letter abbreviations. Because then we're stuck with them forever. If you think it helps the readability of this file, you can make a `local notation` instead.

Reply:
I think maybe the simplest solution will be for me to just change (almost) all the theorems involving `disjoint (M y : set R) ↑I` to just `y ∉ I`. It breaks some of the symmetry between the case of maximal ideals and prime ideals, but it makes them much easier to work with and makes this `M` abbreviation/notation completely unnecessary.